ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/Coverage/compliance.coverage.utils.ts

```diff
@@ -0,0 +1,39 @@
+import { ProgressVariant } from '@patternfly/react-core';
+
+import {
+    ComplianceCheckStatus,
+    ComplianceCheckStatusCount,
+} from 'services/ComplianceEnhancedService';
+
+export function getPassAndTotalCount(checkStats: ComplianceCheckStatusCount[]): {
+    passCount: number;
+    totalCount: number;
+} {
+    let totalCount = 0;
+    let passCount = 0;
+
+    checkStats.forEach((stat) => {
+        totalCount += stat.count;
+        if (stat.status === ComplianceCheckStatus.PASS) {
+            passCount += stat.count;
+        }
+    });
```

Comment:
This works, but a nice pattern for "accumulator" functions is to use the the Array `reduce` method.

Something like
```suggestion
    return checkStats.reduce(
        (acc, stat) => {
            return {
                passCount:
                    stat.status === ComplianceCheckStatus.PASS
                        ? acc.passCount + stat.count
                        : acc.passCount,
                totalCount: acc.totalCount + stat.count,
            };
        },
        { passCount: 0, totalCount: 0 }
    );
```